src/polygonParts/schemas/index.ts

```diff
@@ -9,15 +14,22 @@ import type { EntitiesMetadata, EntityNames, IsSwapQueryParams } from '../models
 
 const polygonPartsEntityNamePatternSchema = z.string().regex(new RegExp(INGESTION_VALIDATIONS.polygonPartsEntityName.pattern));
 
+const findPolygonPartsFeatureSchema = z.object({
+  type: z.literal('Feature'),
+  geometry: polygonSchema.or(multiPolygonSchema).nullable(),
+  properties: z.object({}).passthrough().nullable(),
```

Comment:
doesn't it just define as optional?

Reply:
instead of the 'passthrough'?

Reply:
the geojson schema states that 'properties' must be defined and it is nullable
passthrough is used to enable the use of whatever properties are passed, or else they will be stripped from the parsed response by the zod schema